core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1289,6 +1289,13 @@ public String toString() {
                   + "(alluxio.master.embedded.journal.election.timeout).")
           .setDefaultValue("1s")
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_APPENDER_BATCH_SIZE =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_APPENDER_BATCH_SIZE)
+          .setDescription(" Amount of data that is appended from leader to followers "
+              + "in single batch. Setting higher values might require increasing"
+              + "election timeout due to increased network delay.")
+          .setDefaultValue(1024 * 32)
```

Comment:
this was the original default. Is there a better default?

Reply:
In a separate effort, I'll work on finding a sane default for this and other embedded journal properties.